akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/RouteTest.scala

```diff
@@ -33,7 +34,7 @@ abstract class RouteTest extends AllDirectives with WSTestRequestBuilding {
   implicit def materializer: Materializer
   implicit def executionContext: ExecutionContextExecutor = system.dispatcher
 
-  protected def awaitDuration: FiniteDuration = 3.seconds
+  protected def awaitDuration: FiniteDuration = new JavaTestKit(system).dilated(3.seconds).asInstanceOf[FiniteDuration];
```

Comment:
I think we should use the default Scala testkit definition of dilated here like this:

```scala
import akka.testkit.TestDuration

protected def awaitDuration: FiniteDuration = 3.seconds.dilated
```